client/scss/components/_tags.scss

```diff
@@ -0,0 +1,15 @@
+.tags {
+  @include plain-list();
+  @include font('HNL5');
+  padding-bottom: $vertical-space-unit;
```

Comment:
We could, but we use multiples of it quite frequently as well, for top/bottom padding/margin, so we could wind up with a lorra helper classes. @gestchild reckons?